### ISSUE TITLE
System.InvalidOperationException: Can't search in collection with no items!

### ISSUE THREAD

**Issue by adamhathcock** [AJMitev/FileTypeChecker#14]
Looks like a feature was added but it caused a behavior change on https://github.com/AJMitev/FileTypeChecker/commit/0fcfcc4bd934f14021a5a6aef5cc7fd069532557 for me:

```
System.InvalidOperationException: Can't search in collection with no items!
   at FileTypeChecker.FileTypeValidator.FindMaxScore(IEnumerable`1 matches)
   at FileTypeChecker.FileTypeValidator.FindBestMatch(Stream fileContent, IEnumerable`1 result)
   at FileTypeChecker.FileTypeValidator.GetFileType(Stream fileContent)
```

I think I'm passing a zero byte (unit test) or unknown file type and before it returned null.  Now it's throwing this exception.

This is the expected behavior now?

**Comment by AJMitev** [comment]
Nope, it is a bug with a new implementation. Tomorrow I will release a new version to fix it and add async API

**Comment by AJMitev** [comment]
The issue is resolved in [version 2.0.0](https://www.nuget.org/packages/File.TypeChecker/2.0.0)
